session/session.go

```diff
@@ -11,12 +11,26 @@ import "net/http"
 //	_ = sess.SetString("idp.authenticated", "true")
 //	_ = sess.Save()
 type Provider interface {
-	// GetString returns a session value based on the provided key.
+	// GetString returns a session value based on the provided key. If the key does
+	// not exist, the default or zero value will be returned (i.e, "").
```

Comment:
Should we start returning errors 😉 ? Like `ErrNotFound` and make that available in this pacakge?